src/web/mjs/connectors/CxC.mjs

```diff
@@ -0,0 +1,250 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class CxC extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'cxc';
+        super.label = 'CxC - Content x Creator';
+        this.tags = [ 'webtoon', 'chinese' ];
+        this.url = 'https://cxc.today';
+        this.api = 'https://api.cxc.today';
+        this.uuid = null;
+        this.auth = {
+            accessToken: null,
+            refreshToken: null,
+            expireTime: null
+        };
+        this.requestOptions.headers.set('accept', 'application/json, text/plain, */*');
+        this.requestOptions.headers.set('content-type', 'application/json;charset=utf-8');
+        this.requestOptions.headers.set('referer', this.url);
+        this.requestOptions.headers.set('origin', this.url + '/');
+        this.requestOptions.headers.set('device', 'web');
+        this.requestOptions.headers.set('uuid', this.uuid);
+        this.links = {
+            login: 'https://cxc.today/login'
+        };
+        this.config = {
+            username: {
+                label: 'E-Mail',
+                description: 'E-Mail for login with your CxC account.\nAn account is required to access 18+ content (18禁).\nA fan subscription is required to access VIP content (訂閱粉絲限定).',
+                input: 'text',
+                value: ''
+            },
+            password: {
+                label: 'Password',
+                description: 'Password for login with your CxC account.\nAn account is required to access 18+ content (18禁).\nA fan subscription is required to access VIP content (訂閱粉絲限定).',
+                input: 'password',
+                value: ''
+            }
+        };
+    }
+
+    async _initializeConnector() {
+        await this._getUUID();
+        if (this.config.username.value && this.config.password.value) {
+            await this._login();
+        }
+    }
+
+    async _getMangaFromURI(uri) {
+        await this._refreshAccessToken();
+        var bookAddress = uri.href;
+        var storeName = bookAddress.match(/\/store\/([a-zA-Z0-9_]+)\//)[1];
+        var bookId = bookAddress.match(/\/book\/(\d+)/)[1];
+        let storeRequest = new Request(`${this.api}/store/${storeName}?v=2`, this.requestOptions);
+        let storeData = await this.fetchJSON(storeRequest);
+        var id = `/store/${storeData.data.id}/book/${bookId}`;
+        let bookRequest = new Request(this.api + id, this.requestOptions);
+        let bookData = await this.fetchJSON(bookRequest);
+        if (bookData.data.work_category.id != 1) {
+            throw Error('CxC book type must be comic (漫畫).');
+        }
+        var title = bookData.data.name;
+        return new Manga(this, id, title);
+    }
+
+    async _getMangas() {
+        await this._refreshAccessToken();
+        var books = [];
```

Comment:
General hint : "var" is bad.  Use "let " is variable has to be re-affected or "const" if not. That goest for the whole file